src/index.html

```diff
@@ -11,6 +11,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <label>
+        <div class="search-big" data-qa="big">
+          <input type="text" name="city-search" 
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;" 
+          class="field-big"> 
```

Comment:
fix formatting + let this field be default, add all styles related to it to just field, then rewrite some of them for smaller field

```suggestion
          <input
            type="text"
            name="city-search"
            data-qa="keypress"
            placeholder="Try &quot;Los Angeles&quot;"
            class="field"
          > 
```